internals/controllers/namespace_controller.go

```diff
@@ -293,9 +294,9 @@ func ensureSnsResourcePool(namespace *utils.ObjectContext) error {
 			return err
 		}
 
-		if utils.GetSnsResourcePooled(snsToUpdate.Object) != namespaceResourcePooled && !utils.IsRootResourcePool(snsToUpdate) {
+		if utils.GetNamespaceResourcePooled(namespace) == "true" && !utils.IsRootResourcePool(snsToUpdate) {
```

Comment:
I'm unsure we even need the resourcepool label on the namespace, do we even do anything with it?

Reply:
We use it at the func ComposeChildNamespace but I also think we can delete it and not use it, it's not relevant in my opinion